pumps/mongo_aggregate.go

```diff
@@ -170,12 +177,32 @@ func (m *MongoAggregatePump) Init(config interface{}) error {
 	if m.dbConf.ThresholdLenTagList == 0 {
 		m.dbConf.ThresholdLenTagList = THRESHOLD_LEN_TAG_LIST
 	}
+	// if StoreAnalyticsPerMinute is set to true, the aggregation time will be set to 1.
+	// if not, the aggregation time will be set to the value of the field AggregationTime.
+	// if there is no value for AggregationTime, it will be set to 60.
+	if m.dbConf.StoreAnalyticsPerMinute {
+		m.log.Info("StoreAnalyticsPerMinute is set to true. Setting AggregationTime to 1.")
+		m.dbConf.AggregationTime = 1
+	} else if m.dbConf.AggregationTime < 1 || m.dbConf.AggregationTime > 60 {
+		m.log.Warn("AggregationTime is not set or is not between 1 and 60. Defaulting to 60")
+		m.dbConf.AggregationTime = 60
+	}
 
 	m.connect()
 
 	m.log.Debug("MongoDB DB CS: ", m.dbConf.GetBlurredURL())
 	m.log.Info(m.GetName() + " Initialized")
 
+	// look for the last record timestamp stored in the collection
+	lastTimestampAgggregateRecord, err := getLastDocumentTimestamp(m.dbSession, analytics.AgggregateMixedCollectionName)
+
+	// we will set it to the lastDocumentTimestamp map to track the timestamp of different documents of different Mongo Aggregators
+	if err != nil {
+		m.log.Warn("Unable to get last timestamp from aggregate collection: ", err)
```

Comment:
@mativm02 this Warn is going to show every time we try to cold-start pump since the collection probably is not going to exist.
I'd change the message and the type of the log to Debug.